config/routes.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      resources :users do
+        resources :reservations
+      end
+
+      resources :cities do
+        resources :hotels
+      end
+      resources :room_types, only: [:index]
+
+      resources :hotels
+      resources :cities, only: [:index]
+    end
+  end
+  resources :users, param: :_username
```

Comment:
- It looks like some of the resource commands don't have an only statement, if you forgot it, please add it, if this has a reason, please ignore this. 🐒

Reply:
👋 @vichuge, Thanks for your suggestion. But no problem it works